### ISSUE TITLE
BenchMark: CNN with 5 Conv Layers. Accuracy on FashionMNIST Dataset: 93.15

### ISSUE THREAD

**Issue by Noumanmufc1** [zalandoresearch/fashion-mnist#124]
The model details are as follows:

    Preprocessing by calculating mean and std beforehand.
    Trained using Cross Entropy Loss and Adam Optimizer.
    The initial learning rate is 0.01 which is decreased to 1/4 after every 8 epochs.

    The layers in sequence are:

    Convolutional layer with 32 feature maps of size 3 x 3, stride 1 and pooling 0.
    BatchNorm layer followed by ReLU activation.
    Convolutional layer with 64 feature maps of size 3 x 3, stride 1 and pooling 0
    BatchNorm layer followed by ReLU activation.
    Convolutional layer with 128 feature maps of size 3 x 3, stride 1 and pooling 1
    BatchNorm layer followed by ReLU activation.
    Max Pooling layer of size 2 x 2 and stride 2.
    Convolutional layer with 256 feature maps of size 3 x 3, stride 1 and pooling 0
    BatchNorm layer followed by ReLU activation.
    Convolutional layer with 512 feature maps of size 3 x 3, stride 1 and pooling 0
    BatchNorm layer followed by ReLU activation.
    Max Pooling layer of size 8 x 8 and stride 1.
    Fully connected layer of input size 512 and output size 10.

Accuracy achieved on Fashion MNIST Test Dataset is 93.15%

This network has been implemented in PyTorch. The code can be found [here](https://gist.github.com/Noumanmufc1/60f00e434f0ce42b6f4826029737490a)
